app/stylesheets/bundles/speed_grader.scss

```diff
@@ -733,6 +739,20 @@ body .ui-selectmenu-dropdown:hover .ui-selectmenu-icon, body .ui-state-active .u
           @include fontSize(12px);
         }
       }
+      .vericite_score_container {
+        position: absolute;
+        top: 0;
+        right: 20px;
+      }
+      .vericite_similarity_score {
+        font-size: 9px;
```

Comment:
Why a different in font vs turnitin?  (Otherwise, same as turnitin's version and that should be used.)